var/spack/repos/builtin/packages/catch2/package.py

```diff
@@ -6,11 +6,12 @@
 from spack import *
 
 
-class Catch(CMakePackage):
-    """Catch tests"""
+class Catch2(CMakePackage):
```

Comment:
But this package contains catch 1.* and catch 2.* sources :nerd_face: 
We could split the package, though.

Reply:
Ok, consistent. Good with me then.